js/index.js

```diff
@@ -1,33 +1,89 @@
-// test function for getting data.
-// function getFormData() {
-//     let title = document.getElementsByClassName("title").innerHTML;
-//     let description = document.getElementsByClassName("description").innerHTML;
-    
-//     titleEdit.value = title;
-//     descriptionEdit.value = description;
-// }
+let today = [{
```

Comment:
Move these objects into a separate .json file and import using fetch API (https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch).

Reply:
okh